### PR TITLE
Hotfix Release v8.1.3: Fix misaligned dates and weekdays in calendar

### LINKED ISSUE #2899: [BUG] Calendar dates misaligned with weekday row
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
The calendar dates are not correctly aligned with the weekdays, they are all off by one. 
In the above example, the February 1st looks like it falls on a thursday, when it is in fact on a wednesday.

## Describe how to reproduce the bug

1. Go to https://cookbook.kirby.design/#/home/showcase/calendar
2. See the second example, named 'Wrapped in card + custom buttons integration'

## Which Kirby version was used?
8.1.2

## What was the expected behavior?
Dates should align correctly, e.g. Feburary 1st = wednesday.

## Add any screenshots
image

## Are there any additional context?
Updating any input property causes the calendar to look right. 

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/main/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

### PR DESCRIPTION
**Pull request by RasmusKjeldgaard**
## Which issue does this PR close?

This PR closes #2899

## What is the new behavior?
Old behavior where a locale value was assigned in constructor instead of in OnInit is re-established, as the ngOnChanges hook runs before OnInit and relies on the value. This means it should be configured in the constructor.

This is a hotfix and does not include tests - a follow-up PR will take care of that. 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [x] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [x] Determine if your changes are a fix, feature or breaking-change, and add the matching label to your PR. If it is tooling, dependency updates or similar, add ignore-for-release.
- [x] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [x] Request that the changes are code-reviewed 
- [ ] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

**Review by jakobe — DISMISSED** [review]
LGTM!

**Comment by RasmusKjeldgaard** [comment]
In addition to fixing the bug, this PR also rolls back the documentation from [this PR](https://github.com/kirbydesign/designsystem/pull/2702) on how to supply additional locales via input properties, as the solution has major drawbacks. Instead we want consumers to _provide_ both LOCALE_ID and locale, so they are not forced to manually set the locale via an input on each and every calendar if a non-builtin locale is used.

File: apps/cookbook/src/app/showcase/calendar-showcase/calendar-showcase.component.html [hunk 1]
```diff
@@ -48,57 +48,12 @@ <h4>Events</h4>
...
-    As mentioned in the
-    <code>properties</code>
-    table it's possible to provide additional existing locales or custom ones. But before we take a
-    look at how to provide them, let's take a look at how locales are specified. The selected locale
-    should be injected into the
-    <code>kirby-calendar</code>
-    component. It can be done in the
+    Kirby Calendars built in locales can be configured by providing a
+    <code>LOCALE_ID</code>
+    in for example
```
Comment (jakobe) [inline apps/cookbook/src/app/showcase/calendar-showcase/calendar-showcase.component.html]:
```suggestion
    The locale of the calendar (i.e. day and month names) can be set by providing a
    <code>LOCALE_ID</code>. Usually the locale should be configured for the entire app, typically in
```

File: libs/designsystem/calendar/src/calendar.component.ts [hunk 2]
```diff
@@ -87,7 +87,13 @@ export class CalendarComponent implements OnInit, AfterViewInit, OnChanges {
   @Input() disablePastDates = false;
   @Input() disableFutureDates = false;
   @Input() alwaysEnableToday = false;
-  @Input() locales: { [key: string]: Locale } = {};
+
+  @Input() set locales(locales: { [key: string]: Locale }) {
+    console.warn(
+      `Supplying additional locales to the Kirby CalenderComponent via an input property is deprecated and should not be used. 
+        A future update will allow injecting the locale via a provider instead.`
```
Comment (jakobe) [inline libs/designsystem/calendar/src/calendar.component.ts]:
```suggestion
      `Supplying additional locales to the Kirby Calendar Component via an input property is deprecated and should not be used. 
        A future update will allow injecting additional locales via a provider instead.`
```

**Review by jakobe — COMMENTED** [review]
Looks good, just a few minor suggestions on wording 👌🏻

**Review by jakobe — APPROVED** [review]
🥇

**Pull request merged**